admin/class-convertkit-mm-admin.php

```diff
@@ -142,7 +143,33 @@ public function register_settings() {
 						'tags' => $tags,
 					)
 				);
+			}
+		}
 
+		// Output product to tag mappings.
+		if ( empty( $products ) ) {
+			add_settings_field(
+				'convertkit-empty-mapping',
```

Comment:
Should this be a constant?

Reply:
Agreed - I'd like to move settings to their own class with a constant as we do in the main ConvertKit plugin, and will do this in another PR.